Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Mantid paragons

## Changelog:
- The Klaxxi choose the best of the Mantid culture to be a paragon every 10 years. The Paragons get the Paragon trait, bloodline, are put in slumber and can be awakened by right-clicking later.


## Developer changelog:
- Added all [the historical paragons](https://wow.gamepedia.com/Paragon) (find them in-game).


# How to test:
First, a few commands:
- `event WCRAC.1000` to force the Klaxxi to choose a paragon among all the mantids.
- `event WCRAC.1010` to become a paragon.
- `event WCRAC.1015 <Character ID>` to awaken this paragon.

Check the historical paragons, they should have gamepedia links.

Observe the Klaxxi picking paragons, try to awaken the historical ones and tell your opinion.

Closes #1710.
